atlp-rwanda/destructors-ec-bn
# feat(update user password):update Password after x amount of time

What does this PR do?
Forcing manual password update

Description of Task to be completed?
The task was about requesting a user to change his/her password when it is expired.

How should this be manually tested?
Clone [repo](https://github.com/atlp-rwanda/destructors-ec-bn.git)
Change into the directory destructors-ec-bn
Checkout to this branch: ft-update-password-after-x-time-184637161
In your .env file, create a EXPIRESIN=2,TIMEUNITS=minutes, JOB_SCHEDULING_TIME=* * * * *
Run the command npm install to install all packages
Run npm run dev
you can add a user and verify him.
after 2m as specified in .env you will see the notification from user
What are the relevant pivotal tracker stories?
[184637161](https://www.pivotaltracker.com/n/projects/2633203)
